src/MainWindow.cpp

```diff
@@ -310,13 +313,17 @@ void MainWindow::readSettings()
 
     _statusBarTimeout	 = settings.value( "StatusBarTimeoutMillisec", 3000 ).toInt();
     bool   showTreemap	 = settings.value( "ShowTreemap"	     , true ).toBool();
+    bool   treemapOnSide = settings.value( "TreemapOnSide"	     , true ).toBool();
     QPoint winPos	 = settings.value( "WindowPos"		     , QPoint( -99, -99 ) ).toPoint();
     QSize  winSize	 = settings.value( "WindowSize"		     , QSize (	 0,   0 ) ).toSize();
     _verboseSelection	 = settings.value( "VerboseSelection"	     , false ).toBool();
 
     settings.endGroup();
 
     _ui->actionShowTreemap->setChecked( showTreemap );
+    _ui->actionTreemapAsSidePanel->setChecked( treemapOnSide );
+    treemapAsSidePanel();
```

Comment:
This illustrates that this is not a good name for that new function: In this context, it implies that now the treemap is being displayed as a side panel. But that's not the case; that function determines the main window's splitting orientation. It might be something like `setMainWindowOrientation()` or `setTreemapOrientation()` to make that clear.